openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -47,9 +55,16 @@ def run
             Topic.read_topics(@topics) do |topic, msg_id, msg_hash, redis|
               break if @cancel_thread
 
-              decom_packet(topic, msg_id, msg_hash, redis)
+              if topic =~ /__DECOMINTERFACE/
+                if msg_hash.key?('inject_tlm')
+                  handle_inject_tlm(msg_hash['inject_tlm'])
```

Comment:
Seems like `__DECOMINTERFACE__` is a dedicated topic for inject_tlm. Is there a reason to check the msg_hash for inject_tlm? Is this going to be used for something else?

Reply:
Just inject_tlm for now.  Might be other things in the future.